live/readme.md

```diff
@@ -0,0 +1,26 @@
+# Grafana Live API client
+
+Usage example:
+
+```go
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/grafana/grafana-plugin-sdk-go/live"
+)
+
+func main() {
+	client, err := live.NewClient()
```

Comment:
maybe we can use the package name "grafana" so it is `grafana.NewClient()`  so when we add a method for "QueryData" it is not a big change

Lets not make this too tied to just 'live'

Reply:
Moved to `grafana` - but I feel that the name is too generic, maybe `grafanagrpc` (or sth more beautiful) would be better